src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -52,6 +56,24 @@
     private FluidRegistry fluidRegistry;
     @In
     private InventoryManager inventoryManager;
+    
+    @In
+    private BlockManager blockManager;
+    private Block air;
+    
+    @In
+    private ExtraBlockDataManager extraDataManager;
+    private int flowIx;
+    
+    private static final float fluidPerBlock = 1000;
```

Comment:
I think Checkstyle should complain about this not being capitalized as constant:
```suggestion
    private static final float FLUID_PER_BLOCK = 1000;
```

In addition, you nicely describe how you come to that number in the PR description - would you mind adding this as doc comment here? 🙃